src/test/java/homework_2/exercise1/Exercise1.java

```diff
@@ -0,0 +1,131 @@
+package homework_2.exercise1;
+
+
+import homework_2.AbstractClassForDriver;
+import homework_2.Constants;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class Exercise1 extends AbstractClassForDriver {
+
+    @Test
+    public void testingExercise_1() {
+        // 1. open site
+        driver.get("https://epam.github.io/JDI/index.html");
+
+        //2.Assert Browser title ; 5.Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        // 3.Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("epam");
+        driver.findElement(By.id("password")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+
+        WebElement loginedUserName = new WebDriverWait(driver, 15)
+                .until(ExpectedConditions.presenceOfElementLocated(By.id("user-name")));
+
+        // 4.Logined user's name is display correctly
+        assertEquals(loginedUserName.getText(), "PITER CHAILOVSKII");
+
+        //6.Menu buttons are displayed and have proper texts "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+        List<WebElement> headerItems = new ArrayList<>();
+        for (int i = 1; i <= 4; i++) {
+            headerItems.add(driver.findElement(By.xpath(
+                    "//ul[@class='uui-navigation nav navbar-nav m-l8']/li[" + i + "]")));
+        }
+        for (WebElement el: headerItems) {
+            assertTrue(el.isDisplayed());
+        }
+        assertEquals(headerItems.get(0).getText(), "HOME");
+        assertEquals(headerItems.get(1).getText(), "CONTACT FORM");
+        assertEquals(headerItems.get(2).getText(), "SERVICE");
+        assertEquals(headerItems.get(3).getText(), "METALS & COLORS");
+
+        //7. 4 Images are displayed
+        WebElement imgPractise = driver.findElement(By.xpath(
+                "//div[@class='col-sm-3']//span[@class='icons-benefit icon-practise']"));
```

Comment:
I think, that current **xpath** could be simplified. 
Why do you decide use full list of the classes in the xpath?